docs/manual-bridging.md

```diff
@@ -75,21 +83,56 @@ Note: The bridge only supports standard ERC-20 tokens, so make sure your token a
+- The transaction will first transfer and burn (except for wIMX) the specified amount of ERC-20 from the user to the bridge and perform a cross-chain call to the L1 to unlock a corresponding amount of tokens to the intended receiver address.
+- This process can take about 20 minutes to complete on the destination chain. You can track the progress of this cross-chain call by going to Axelarscan ([mainnet](https://axelarscan.io/), [testnet](https://testnet.axelarscan.io/)) and providing the transaction hash in the search field at the top right.
+- Additionally, for newly mapped tokens or tokens for which bridge flow rate parameters have not been configured, the token will be queued for an additional 24 hours before it can be claimed by the user. See [here](https://github.com/immutable/zkevm-bridge-contracts/blob/main/docs/high-level-architecture.md#withdrawal-delays) for more details. After the delay has elapsed a user will have to claim their funds on the L1 bridge by calling `finaliseQueuedWithdrawal()` on the [root bridge proxy contract](https://github.com/immutable/zkevm-bridge-contracts/blob/main/docs/high-level-architecture.md#withdrawal-delays), and providing the `receiver` address used during the withdrawal and an index number of the queued withdrawal (starting from 0 for the first queued withdrawal for the specific recipient).
+
+### Withdrawing IMX
+- Navigate to the Child bridge contract on Blockscout ([mainnet](https://explorer.immutable.com/address/0xBa5E35E26Ae59c7aea6F029B68c6460De2d13eB6), [testnet](https://explorer.testnet.immutable.com/address/0x0D3C59c779Fd552C27b23F723E80246c840100F5))
+- Select Contract → “Write as Proxy”.
+- Execute the `withdrawIMX()` function, providing the following parameters:
+    - `amount`: Amount to transfer in base unit e.g. 1 IMX would be represented as `100000000000000`
+    - `native IMX` : This amount should be the sum of two things 
+      - Amount to transfer in IMX (see `amount` above)
+      - Bridge fee estimate in ETH (see [Estimating Bridge Fee](#estimating-bridge-fee) for details)
+    - **Note:** The `withdrawIMX()` function will transfer funds to the same address on L1 as the sender executing this transaction on L2. If however, you’d like to deposit to a different address use the `withdrawToIMX`function, which will take an additional parameter `receiver` which is the address of the receiver on L1.
+- This process can take ~20minutes to be completed on the destination chain. You can track the progress of this cross-chain call by going to Axelarscan ([mainnet](https://axelarscan.io/), [testnet](https://testnet.axelarscan.io/)) and providing the transaction hash in the search field at the top right.
+- Once completed, the funds will be available in the recipient's address on L1. Note that whether you deposit native IMX using this method or wrapped IMX via the ERC-20 token deposit method, the same IMX token will be received on L1.
```

Comment:
This would also be potentially rate-limited and would involve a finalisation txn on Ethereum?